lib/tasks/migrate_labware_audits.rake

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+namespace :labware_events do
+  # to run this use bundle exec rake END_DATE=2020-09-24
+  desc "Move the labware audits to the events warehouse"
+  task create: :environment do |_t|
+    # any audits before this date will have events created
+    end_date = DateTime.parse(ENV['END_DATE'])
+
+    # labwares
+    audits = Audit.includes(:auditable).where("auditable_type = 'Labware' and created_at <= :date", date: end_date)
+
+    Rails.logger.info("LABWHERE EVENTS:  There are #{audits.count} audits to be processed")
+
+    events_created = 0
+
+    audits.each do |audit|
+      event = Event.new(labware: audit.auditable, audit: audit)
```

Comment:
find_each is probably safer here, pulls them down in batches of 1000, which is useful memory wise if there are a lot.

Reply:
Also, logging the audit id here can be useful as a progress indicator if things fall over. Means you know where things got to and can resume from there.